matchbox_socket/src/webrtc_socket/native.rs

```diff
@@ -427,7 +428,7 @@ async fn create_rtc_peer_connection(
             urls: ice_server_config.urls.clone(),
             username: ice_server_config.username.clone().unwrap_or_default(),
             credential: ice_server_config.credential.clone().unwrap_or_default(),
-            ..Default::default()
+            credential_type: RTCIceCredentialType::Password,
```

Comment:
We should probably only set this if credential has a value?

Also, can you put a comment explaining why we have to do this workaround and set a deprecated field.

Reply:
I made the field conditional, but I'm not sure where you want the depricated field?